### ISSUE TITLE
Build error related to PLUGINLIB_EXPORT_CLASS

### ISSUE THREAD

**Issue by andreucm** [JavierRubia/Garrinator#3]
The following build error appears, when compiling 5c01f4a19a8f757ae8baa0bcbe967ebf4533c032 (branch https://github.com/JavierRubia/Garrinator/tree/add_joint_state_controller) : 

```
/home/andreu/dev/ros_ws/src/Garrinator/hardware_interface/src/odrive_hw_if.cpp:281:94: error: expected constructor, destructor, or type conversion at end of input
 PLUGINLIB_EXPORT_CLASS(garrinator_hardware_interface::OdriveHwIf, hardware_interface::RobotHW)
                                                                                              ^
Garrinator/hardware_interface/CMakeFiles/garrinator_hardware_interface.dir/build.make:62: recipe for target 'Garrinator/hardware_interface/CMakeFiles/garrinator_hardware_interface.dir/src/odrive_hw_if.cpp.o' failed
make[2]: *** [Garrinator/hardware_interface/CMakeFiles/garrinator_hardware_interface.dir/src/odrive_hw_if.cpp.o] Error 1
CMakeFiles/Makefile2:9540: recipe for target 'Garrinator/hardware_interface/CMakeFiles/garrinator_hardware_interface.dir/all' failed
make[1]: *** [Garrinator/hardware_interface/CMakeFiles/garrinator_hardware_interface.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
```

It seems to be related with PLUGINLIB_EXPORT_CLASS. I've checked that both constructors exist, derived and base class. 

Any idea ?

**Comment by JavierRubia** [comment]
here `https://github.com/JavierRubia/Garrinator/blob/5c01f4a19a8f757ae8baa0bcbe967ebf4533c032/hardware_interface/include/odrive_hw_if.h#L31` seems missing the `;` and dont know why is there the `{}`

Edit: that is not affecting to the error, I've tested it and compiled it, but the same error shows on, where is the constructor of the class: `OdriveHwIf()` ?

**Comment by andreucm** [comment]
Fixed at 169d599614c9695c2654a151a775c9edeaac6dcb
coming back to the previous situation.
